lib/fastlane/actions/swiftlint.rb

```diff
@@ -5,7 +5,21 @@ def self.run(params)
         if `which swiftlint`.to_s.length == 0 and !Helper.test?
           raise "You have to install swiftlint using `brew install swiftlint`".red
         end
-        Actions.sh("swiftlint")
+
+        command_prefix = [
+          'cd',
+          File.expand_path('.').shellescape,
+          '&&'
+        ].join(' ')
+        swiftlint_args = "> #{params[:report_file]}"
```

Comment:
should this check for the presence of the param before assigning the var? what happens if there is no `:report_file` param?